picard/ui/mainwindow.py

```diff
@@ -381,6 +382,10 @@ def create_actions(self):
         self.browser_lookup_action.setEnabled(False)
         self.browser_lookup_action.triggered.connect(self.browser_lookup)
 
+        self.more_results_action = QtGui.QAction(_(u"Display more results"), self)
+        self.more_results_action.setStatusTip(_(u"Display more results"))
```

Comment:
No it's not intended to be the same (that'll not be a good status tip :D ). I had put it temporarily, so thanks for the reminder. I'll update it.